micro_http/src/connection.rs

```diff
@@ -361,6 +361,11 @@ impl<T: Read + Write> HttpConnection<T> {
     pub fn pop_parsed_request(&mut self) -> Option<Request> {
         self.parsed_requests.pop_front()
     }
+
+    /// Returns true if there are bytes waiting to be written into the stream.
+    pub fn is_outgoing(&self) -> bool {
```

Comment:
Please use a more proper name for this function. **Outgoing** usually indicates direction, but in this specific case it is about state.

Reply:
How about `pending_write`?

Reply:
Sounds good enough for me.